ebml/EbmlDate.h

```diff
@@ -57,6 +57,17 @@ class EBML_DLL_API EbmlDate : public EbmlElementDefaultSameStorage<std::int64_t>
 
     filepos_t ReadData(IOCallback & input, ScopeMode ReadFully = SCOPE_ALL_DATA) override;
 
+  protected:
+    static std::int64_t EpochToEBML(std::int64_t epoch)
```

Comment:
So far we're only using all-uppercase `EBML` for preprocessor defines and CamelCase `EbmlWhatever` for classes & functions. Can we please continue doing so, for consistency's sake (`EbmlToEpoch` & `EpochToEbml`)? Thanks.

Reply:
OK, done.